doc/tutorials/core/basic_geometric_drawing/basic_geometric_drawing.markdown

```diff
@@ -145,7 +145,7 @@ Explanation
             of size **(w/4.0, w/16.0)**
         -   The ellipse is rotated **angle** degrees
         -   The ellipse extends an arc between **0** and **360** degrees
-        -   The color of the figure will be **Scalar( 255, 255, 0)** which means blue in RGB value.
+        -   The color of the figure will be **Scalar( 255, 0, 0)** which means blue in RGB value.
```

Comment:
If so this should be changed in a lot of places I think because as I understand the RGB values are reversed in OpenCV as BGR. Maybe that way is better.